dash.js

```diff
@@ -715,23 +695,10 @@ var MyDash = GObject.registerClass({
                 newApps.push(favorites[id]);
         }
 
-        // We reorder the running apps so that they don't change position on the
-        // dash with every redisplay() call
         if (settings.get_boolean('show-running')) {
-            // First: add the apps from the oldApps list that are still running
```

Comment:
This workaround needs to be ported and re-added to avoid icon reordering issues.

(or perhaps a new workaround needs to be created)

Reply:
Ok I think @micheleg fixed this.